developer-docs-site/docs/concepts/aptos-token.md

```diff
@@ -0,0 +1,172 @@
+---
+title: "Aptos Token"
+id: "aptos-token"
+---
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Aptos Token
+
+For digital assets, Aptos provides two Move modules:
+- The [`token.move`](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token/sources/token.move) for the token standard. The token standard covers non-fungible (NFT), semi-fungible and **fungible non-decimal** token.
+- The [`coin.move`](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/coin.move) for the coin standard. The coin standard specifies the **fungible decimal** token. 
+
+## Token vs coin standards
+
+The coin standard is separated out into its own Move module to ensure that:
+
+- The coin standard can be used to create a token with decimal and with minimal metadata. 
+- The coin module remains a part of the Aptos core framework and be used for the gas currency, thereby enhancing the core functionality of the Aptos framework.
+
+The token standard, on the other hand, is deployed as a separate package at the Aptos blockchain address `0x3`. The token standard is designed to create an NFT or a semi-fungible or a fungible non-decimal token, with rich metadata and functionalities. A token definition of this type can be iterated rapidly to respond to the platform and user requirements. 
+
+:::tip Aptos token standard
+This document presents the proposed Aptos token standard. The Aptos implementation for core NFTs or tokens can be found in [token.move](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token/sources/token.move). Also see the tutorial [Your First NFT](/tutorials/your-first-nft.md).
+
+The coin standard is presented in a different document.
+:::
+
+### Overview of NFT
```

Comment:
feels like this should be another doc rather than here. I think that's what was discussed in the chat thread?